.github/workflows/build-docker-admin.yml

```diff
@@ -9,33 +9,11 @@ on:
 jobs:
   docker-build-admin:
     if: ${{ github.event.workflow_run.conclusion == 'success' }}
-    runs-on: ubuntu-latest
-    steps:
-
-    - name: Checkout
-      uses: actions/checkout@v4
-
-    - name: Docker Login
-      uses: docker/login-action@v3
-      with:
-        registry: fruoccopublic.azurecr.io
-        username: fruoccopublic
-        password: ${{ secrets.DOCKER_PASSWORD }}
-
-    - name: Set up Docker Buildx
-      uses: docker/setup-buildx-action@v3
-
-    - name: Get current date
-      id: date
-      run: echo "date=$(date +'%Y-%m-%d')" >> $GITHUB_OUTPUT
-
-    - name: Build and push the Admin Docker image
-      uses: docker/build-push-action@v5
-      with:
-        context: .
-        file: docker/Frontend.Dockerfile
-        push: true
-        cache-from: type=registry,ref=fruoccopublic.azurecr.io/rag-adminwebapp:latest
-        tags: |
-          fruoccopublic.azurecr.io/rag-adminwebapp:latest
-          fruoccopublic.azurecr.io/rag-adminwebapp:${{ steps.date.outputs.date }}_${{ github.run_number }}
+    uses: ./.github/workflows/build-docker.yml
```

Comment:
Should we merge these into a single workflow like you have with the "without push" workflow?